scripts/common/common.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+set -e
+
+# Module system
+function log() {
+    echo -e "\e[32m\"[DEPLOY LOG] $*\"\e[0m"
+}
+
+function pull_experiment() {
+  GIT_URL=$1
+  HASH_COMMIT=$2
+  FOLDER=$3
+  log "downloading source code from $GIT_URL to $FOLDER"
+  git clone -q "$GIT_URL" "$FOLDER"/
+  cd "$FOLDER" || exit
```

Comment:
set -e alraeady does the || (it might be my issue i can't remember, but since you are changing it might aswell.)

Reply:
The `||` was already there

Reply:
Remove it if you want

Reply:
I wish... but Pycharm then gives me anxiety with its yellow squiggle :<